vnf.scad

```diff
@@ -827,5 +827,130 @@ module vnf_validate(vnf, size=1, show_warns=true, check_isects=false) {
     color([0.5,0.5,0.5,0.5]) vnf_polyhedron(vnf);
 }
 
+// Section: VNF transformations
+//
+// Function: vnf_halfspace(halfspace, vnf)
+// Usage:
+//   vnf_halfspace([a,b,c,d], vnf)
+// Description:
+//   returns the intersection of the VNF with the given half-space.
+// Arguments:
+//   halfspace = half-space to intersect with, given as the four
+//   coefficients of the affine inequation a*x+b*y+c*z+d ≥ 0.
```

Comment:
We have a plane definition used in geometry.scad where `[A,B,C,D]` is `Ax+By+Cz=D`.  Can we use this for consistency?  The normal of the plane would specify which side of the plane (or halfspace) to use.

Reply:
Also, all arguments in the Arguments: block have to be one-line-per-argument with the syntax `name = description`